kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/DefaultSharedIndexInformerTest.java

```diff
@@ -251,7 +250,7 @@ public void onDelete(Pod oldObj, boolean deletedFinalStateUnknown) { }
     factory.startAllRegisteredInformers();
 
     // Wait for resync period of pass
```

Comment:
You should remove this comment ... as now Thread.sleep is passed 500 as value rather than the RESYNC_PERIOD value.

Reply:
Or maybe keep it as Thread.sleep(RESYNC_PERIOD) .. whatever is better

Reply:
The test needs to be refactored to remove any Thread.sleep invocation.
For the scope of the current PR, maybe just remove _misleading_ the comment.